javascript/apps/taiga/src/app/shared/comments/components/comment-detail/comment-detail.component.ts

```diff
@@ -76,32 +74,24 @@ export class CommentDetailComponent {
 
   public deleteConfirm(): void {
     this.showDeleteCommentConfirm = true;
-    this.highlightComment.next(this.comment.id);
+    this.highlightComment.emit(true);
   }
 
   public changeCommentOptionsState(state: boolean): void {
     this.commentOptionsState = state;
     if (!state) {
       this.showDeleteCommentConfirm = false;
-      this.highlightComment.next(undefined);
+      this.highlightComment.emit(false);
       this.unsetDelete$.next();
```

Comment:
I think this is not being used

Reply:
I just did the refactor of this part but seems to me that is being used because it emit its value to its parent and sets the `higlighted` value to false (removes the highlight)